postgres/tests/test_statements.py

```diff
@@ -917,6 +917,15 @@ def wait(conn):
         assert bobs_query is not None
         assert blocking_bobs_query is not None
 
+        non_client_backend = [q for q in event['postgres_activity'] if q.get('backend_type', 'client backend') != 'client backend']
```

Comment:
cant we use this same method herE to check the version?

```
check.version.compare(V10) >= 10
```
then just do the assert for versions 10+?